src/nft/pages/collection/index.tsx

```diff
@@ -216,12 +220,12 @@ const Collection = () => {
   )
 }
 
+// TODO: Remove when assets query is migrated to Apollo
 // The page is responsible for any queries that must be run on initial load.
 // Triggering query load from the page prevents waterfalled requests, as lazy-loading them in components would prevent
 // any children from rendering.
 const CollectionPage = () => {
   const { contractAddress } = useParams()
-  useLoadCollectionQuery(contractAddress)
```

Comment:
do you not want to preload this anymore?

Reply:
This wasn't really preloading as it's run directly before the pageload so there wasn't any time saved.